lib/utils/fileSorting.ts

```diff
@@ -65,7 +65,7 @@ export function sortNodes(nodes: readonly INode[], options: FilesSortingOptions
 		// 3: Use sorting mode if NOT basename (to be able to use display name too)
 		...(sortingOptions.sortingMode !== FilesSortingMode.Name ? [(v: INode) => v[sortingOptions.sortingMode]] : []),
 		// 4: Use display name if available, fallback to name
-		(v: INode) => basename(v.attributes?.displayname || v.basename),
+		(v: INode) => basename(v.displayname || v.basename),
```

Comment:
```suggestion
		(v: INode) => basename(v.displayname || v.attributes?.displayname || v.basename),
```

We need to keep the old behavior as a fallback as there are apps that still use old library versions, so they could pass a Node without `displayname` property to the files app which uses this function then.

This should still silence the warning, because `Node.displayname` is always set for modern library versions, only Nodes passed from old apps will cause the fallback.